docs/source/pillars/information_governance.md

```diff
@@ -156,64 +295,200 @@ _The ability of the TRE organisation to manage projects effectively._
+    This would include ownership, data lifecycle, contracts, risk assessments and other quality data.
+    This is likely to already exist within the wider organisation but may require augmenting for the TRE.
+  - Mandatory
+```
+
+**Study Register:** A centralised record or database that tracks and manages information about research studies and projects.
 
 ```{list-table}
 :header-rows: 1
-:name: tab-onboarding-members
+:name: tab-ig-study-register
 
 * - Statement
   - Guidance
   - Importance
-* - You must have clear onboarding processes in place for all roles within your TRE organisation.
-  - This may include all members signing role-specific terms of use, and completing role specific training.
+* - You must keep a complete record of all the research studies and projects within the TRE current and past.
+  - The study register should contain all data related to a study including a reference to data assets, members (researchers, owners etc.) and any compliance activities required.
   - Mandatory
+```
+
+## Member Accreditation
+
+Ability to ensure that people with access to data are correctly identified and they are suitably qualified.
+
+**Identity Verification:** The process of confirming or authenticating the identity of individuals or entities, often through the verification of personal information, credentials, or biometric data.
+
+```{list-table}
+:header-rows: 1
+:name: tab-ig-identity-verification
+
+* - Statement
+  - Guidance
+  - Importance
 * - You must have a robust method for identifying accredited members of your TRE organisation, prior to their accessing of sensitive data.
   - This may include multi-factor authentication (MFA), ID checks or email/phone verification.
   - Mandatory
 ```
 
-### Training management and delivery
+**User Onboarding:** The process of introducing and integrating researchers and data consumers onto a TRE's systems, processes, including training, access provisioning, and orientation.
+
+```{list-table}
+:header-rows: 1
+:name: tab-ig-user-onboarding
+
+* - Statement
+  - Guidance
+  - Importance
+* - You must have clear onboarding processes in place for all roles within your TRE organisation.
+  - This may include all members signing role-specific terms of use and completing role specific training.
+  - Mandatory
+```
+
+**Identity and Access Management Services:** Govern and control user identities, access privileges, authentication, and authorization within an organisation.
 
 ```{list-table}
 :header-rows: 1
-:name: tab-training-management-delivery
+:name: tab-ig-identity-access-management-services
+
+* - Statement
+  - Guidance
+  - Importance
+* - You must have a set of services to manage access to resources based on identity.
+  - This will include a security model for role based access with technical controls to ensure the principle of least privilege is enforced.
+  - Mandatory
+* - The data asset owner must have control of who has access to which assets.
```

Comment:
What does "ownership" mean? Are we talking about the Data Owner? Typically TRE operators might be Controllers or Processors but are not necessarily owners of the data itself.

Reply:
Good point, I think it depends on what _control_ means. Does it mean the data owner has direct technical control of access via TRE permissions, or is it higher level governance in the sense that the data owner _controls_ access by telling the data controllers/processors who should be allowed access or what criteria should be used to decide access, and the controller/processor must implement that by the appropriate technical means?

Reply:
Thoughts @machintim ?

Reply:
```suggestion
* - You must not give anyone access to datasets without agreement from the Data Controller.
```